13/abidmunirmalik/directors.py

```diff
@@ -0,0 +1,82 @@
+import csv
+from collections import defaultdict, namedtuple, Counter
+
+MOVIE_DATA = 'movie_metadata.csv'
+NUM_TOP_DIRECTORS = 20
+MIN_MOVIES = 4
+MIN_YEAR = 1960
+
+def get_movies_by_director():
+    '''Extracts all movies from csv and stores them in a dictionary
+    where keys are directors, and values is a list of movies (named tuples)'''
+    Movies  = namedtuple('Movies', ['title','year','score'])
```

Comment:
namedtuple creates a class with the properties title year and score. So strictly spoken it is not "Movies" but "Movie", because each new Movie() is only one movie instance and not a collection of movies